### ISSUE TITLE
Strange German Translation in settings

### ISSUE THREAD

**Issue by bdorer** [nextcloud/notes-ios#60]
In the settings the last checkbox is named "interner Redakteur". In English that means probably "internal editor". Never the less I'm not able to see any diffrence weather I turn it on or off. Please clarify that with a better translation or with a helpful text.

**Comment by i2h3** [comment]
You are right, the translation is misleading. Switching it should make you see in the note editing that a native text editor is used instead of the web-based one.

The German translation, including this odd text, were updated in c75c1a0. It was just not released yet.
